sound/soc/sof/topology.c

```diff
@@ -1532,7 +1532,7 @@ static int sof_widget_ready(struct snd_soc_component *scomp, int index,
 	if (ret < 0) {
 		dev_err(scomp->dev,
 			"error: failed to add widget id %d type %d name : %s stream %s\n",
-			tw->shift, swidget->id, tw->name,
+			swidget->comp_id, swidget->id, tw->name,
```

Comment:
@yongzhi1 is the comp_id even useful for anything? Why not remove this altogether?

Reply:
Ack, let's remove it :)